common/djangoapps/student/models.py

```diff
@@ -148,9 +148,9 @@ def anonymous_id_for_user(user, course_id, save=True):
             log.error(
                 u"Stored anonymous user id %r for user %r "
                 u"in course %r doesn't match computed id %r",
+                anonymous_user_id.anonymous_user_id,
                 user,
                 course_id,
-                anonymous_user_id.anonymous_user_id,
                 digest
```

Comment:
yeah, we could pass a dictionary: http://www.diveintopython.net/html_processing/dictionary_based_string_formatting.html

But I honestly don't think we gain much by doing it